www/src/data/sidebars/doc-links.yaml

```diff
@@ -414,8 +414,6 @@
               items:
                 - title: "@reach/router and Gatsby"
                   link: /docs/reach-router-and-gatsby/
-                - title: Linking and Prefetching with Gatsby*
-                  link: /docs/linking-and-prefetching-with-gatsby/
```

Comment:
is there a redirect needed for the removed stub?

Reply:
The stub was not referenced anywhere else in the codebase. So I don't believe so.

Reply:
Ok. I found the page only with exact keywords search https://www.google.com/search?q=gatsby+linking+and+prefetching